_site/assets/css/main.css

```diff
@@ -1 +0,0 @@
on .logo-wrapper .project-logo{max-width:180px;margin-top:.5rem}nav ul li a.active,nav ul li a.active:hover,nav ul li a.active:focus,nav ul li a.active:visited{font-weight:400;border-bottom:4px solid #F5891F;color:#085870}@media screen and (max-width: 1024px){nav ul li a.active,nav ul li a.active:hover,nav ul li a.active:focus,nav ul li a.active:visited{border:none}}nav .nav-mobile{top:2.5rem}nav .nav-mobile a,nav ul.nav-list li a{text-transform:uppercase;color:#085870}nav .nav-mobile a:hover,nav .nav-mobile a:focused,nav .nav-mobile a:visited,nav ul.nav-list li a:hover,nav ul.nav-list li a:focused,nav ul.nav-list li a:visited{color:#085870}nav .nav-mobile a.button-cta,nav ul.nav-list li a.button-cta{text-transform:uppercase;color:#fff;line-height:2.5;padding:2px 10px 0 10px}#nav-toggle span,#nav-toggle span:before,#nav-toggle span:after{background:#085870}.homepage-hero-band{background:#085870;background:-moz-linear-gradient(-45deg, #085870 0%, #085870 50%, #97d4e8 100%);background:-webkit-linear-gradient(-45deg, #085870 0%, #085870 50%, #97d4e8 100%);background:linear-gradient(135deg, #085870 0%, #085870 50%, #97d4e8 100%);filter:progid:DXImageTransform.Microsoft.gradient( startColorstr='#085870', endColorstr='#97d4e8',GradientType=1 );margin:-4rem -13rem 0 -13rem}@media screen and (max-width: 1170px){.homepage-hero-band{margin:0 -6rem;margin-top:-2rem}}@media screen and (max-width: 480px){.homepage-hero-band{margin:0 -2rem}}.homepage-hero-band .wrapper{background-image:url("/assets/images/homepage/hero_background.svg");background-repeat:repeat-x;background-size:auto;background-position:left;padding:6rem 13rem 8rem 13rem}@media screen and (max-width: 1024px){.homepage-hero-band .wrapper{background-size:cover}}@media screen and (max-width: 1170px){.homepage-hero-band .wrapper{padding:4rem 6rem 6rem 6rem}}@media screen and (max-width: 480px){.homepage-hero-band .wrapper{padding:4rem 2rem 6rem 2rem}}.homepage-hero-band h1,.homepage-hero-band h2{color:#fff}@media screen and (min-width: 768px){.homepage-hero-band h1{font-size:6rem;line-height:6rem}}.homepage-hero-band h2{font-family:'Josefin Sans', sans-serif;text-transform:uppercase;font-size:1.2rem;line-height:1;max-width:42rem}.homepage-hero-band a.button-cta,.homepage-hero-band a.button-cta:hover,.homepage-hero-band a.button-cta:focus,.homepage-hero-band a.button-cta:visited{border-color:#fff;margin-left:0}.tabs{display:flex;flex-wrap:wrap;margin:2rem 0}.tabs label{order:1;display:block;padding:1rem 2rem;margin-right:0.2rem;cursor:pointer;background-color:#daedf5;font-weight:700;transition:background ease 0.2s}.tabs .tab{order:99;flex-grow:1;width:100%;display:none;padding:1rem;background-color:#f6f5f5}.tabs input[type="radio"]{display:none}.tabs input[type="radio"]:checked+label{background-color:#f6f5f5}.tabs input[type="radio"]:checked+label+.tab{display:block;overflow:scroll}@media (max-width: 45em){.tabs .tab,.tabs label{order:initial}.tabs label{width:100%;margin-right:0;margin-top:0.2rem}}
```

Comment:
is this deleted file on purpose?

Reply:
not on purpose, trying to revert, thank you for spotting